.github/workflows/ci.yaml

```diff
@@ -0,0 +1,21 @@
+name: CI
+on: [push]
+jobs:
+  test:
+    strategy:
+      fail-fast: false
```

Comment:
I'm pretty sure Travis is set to fail-fast in it's config file. Want to pick a value to use in both?